views/js/controller/list/index.js

```diff
@@ -322,6 +332,22 @@ define([
         }
     }
 
+    function handleLoadMore() {
+        debugger;
```

Comment:
-  [ ] Do not forget to remove this. 

As a suggestion, please use @FIXME and @TODO annotations to mark the code you need to remove after. Example: 

```suggestion
        debugger; //@FIXME Remove this before merged
```
Before you merge the PR you can easily find those places.

Reply:
I think, @adriloma still working on this task and will remove this asap 🙂

Reply:
I know, but it is safer to only push test code with FIXME or TODO annotations to avoid merging by mistake.